File: app/src/main/java/omniex/nl/omniex/ui/app/order/payment/OrderPaymentView.java
package omniex.nl.omniex.ui.app.order.payment;

import java.util.List;

import nl.omniex.omniexshopping.data.model.address.Address;
import nl.omniex.omniexshopping.ui.base.BaseView;

public interface OrderPaymentView extends BaseView {

    void onPaymentAddressesFetched(List<Address> addressList);

    void onPaymentMethodSet();
}

File: app/src/main/java/omniex/nl/omniex/ui/app/order/payment/OrderPaymentView.kt
package omniex.nl.omniex.ui.app.order.payment

import omniex.nl.omniex.data.model.address.Address
import omniex.nl.omniex.ui.base.BaseView


interface OrderPaymentView : BaseView {

    fun onPaymentAddressesFetched(addressList: List<Address>)

    fun onPaymentMethodSet()
}
